docs/getting-started/prerequisites.md

```diff
@@ -0,0 +1,237 @@
+# Setting up the prerequisites
+
+This guideline includes the steps to install the prerequisites with specified versions for the Carbyne Stack. All installation procedures are done with the Terminal application of Ubuntu. **Please open the Terminal application before you proceed.**
```

Comment:
I'd suggest to replace the whole paragraph with:

```
This guide describes how to install the prerequisites required to deploy Carbyne Stack.
```

We can omit the references to using a "terminal". People who do not know how to work with the command line are not part of the target audience of these tutorials.

Reply:
I have replaced the paragraph with this line. I guess this is less work for me with this group of target audiences.